runtime/oti/j9nonbuilder.h

```diff
@@ -4058,6 +4058,25 @@ typedef struct J9JITConfig {
 #endif /* J9VM_OPT_JITSERVER */
 } J9JITConfig;
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+typedef BOOLEAN (*criuHookFunc)(struct J9JavaVM *vm, void *userData, j9object_t obj, BOOLEAN isRestoreHook);
+typedef struct J9ObjectRecord {
+	j9object_t object;
+} J9ObjectRecord;
+typedef struct J9InternalHookRecord {
```

Comment:
this should record if its restore hook or not

Reply:
Added.